### DISCUSSION TITLE
Audio quality seems lower than in other apps

### DISCUSSION THREAD

**Discussion by gaker19sc** [z-huang/music#1206] [General]
When there is a lot going on, you start to notice the audio getting muddy. I tried it on every speaker I own and it seems to be the most noticable on headphones, but it's also annoying on other speakers. I've seen a reddit thread about this with no responses. It's a bummer because InnerTune is basically perfect, this is the only thing keeping it away from being my number one music app. The muddy audio is not present on similar apps like Simpmusic, Blackhole or even YouTube Music. So I really don't know what's causing it. Messing with the equalizer doesn't seem to help and the audio quality is already set to high.

**Comment by RGBCube** [comment]
I also have the same experience. Switched from ViMusic and the audio is noticeably lower in quality even in High mode for Audio Quality. And yes, I've cleared all cache.

**Comment by RGBCube** [comment]
Hmm, after logging out of my YouTube Music account and purging all cache, quality is better. Are you logged in to your YouTube account?

**Comment by hewiofhiowa** [comment]
Check the details and see what codec it's using if it's 251 opus then that should be the highest quality YouTube offers
